src/test/java/com/alipay/oceanbase/hbase/HTableTestBase.java

```diff
@@ -3435,26 +3435,7 @@ public void testFamilyBlank() throws Exception {
         } catch (IllegalArgumentException e) {
             Assert.assertTrue(e.getMessage().contains("family is empty"));
         }
-
-        Get get = new Get(key.getBytes());
-        get.addColumn(Bytes.toBytes(""), null);
```

Comment:
why remove this test?

Reply:
These cases are now supported and will no longer throw errors.